apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -722,31 +721,16 @@ def read(
         _util.check_unpartitioned(partitions)
         self._check_open_read()
 
-        handle = self._handle._handle
-
-        context = handle.context()
-        if platform_config is not None:
-            config = context.tiledb_config.copy()
-            config.update(platform_config)
-            context = clib.SOMAContext(config)
-
-        sr = clib.SOMADataFrame.open(
-            uri=handle.uri,
-            mode=clib.OpenMode.read,
-            context=context,
-            column_names=column_names or [],
+        # TODO: batch_size
```

Comment:
Please file a GitHub issue for tracking this, and include its URL here as a comment

Reply:
https://github.com/single-cell-data/TileDB-SOMA/issues/1175